src/code/z_kankyo.c

```diff
@@ -860,46 +892,53 @@ void Environment_Update(GlobalContext* globalCtx, EnvironmentContext* envCtx, Li
             }
         }
 
-        func_800766C4(globalCtx); // increments or decrements unk_EE[1] depending on some condition
-        func_80075B44(globalCtx); // updates bgm/sfx and other things as the day progresses
+        Environment_UpdateRain(globalCtx);
+        Environment_PlayTimeBasedSequence(globalCtx);
 
-        if (((void)0, gSaveContext.nextDayTime) >= 0xFF00 && ((void)0, gSaveContext.nextDayTime) != 0xFFFF) {
+        if (((void)0, gSaveContext.nextDayTime) >= 0xFF00 && ((void)0, gSaveContext.nextDayTime) != NEXT_TIME_NONE) {
```

Comment:
been a minute since ive looked at any of this stuff

but at first glance, the magic number seems better to me than doing math that happens to end up at the number. it doesnt really make much sense. I see your point about being able to locate it, but I dont think thats enough for me